app/src/main/java/idrisadetunmbi/githubusers/GithubUserAdapter.java

```diff
@@ -0,0 +1,65 @@
+package idrisadetunmbi.githubusers;
+
+import android.content.Intent;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import java.util.List;
+
+import idrisadetunmbi.githubusers.models.GithubUser;
+
```

Comment:
One the next week's task is to add lint and style checks to your code, but before then please try and generate comments for every section of your code